CONTRIBUTING.md

```diff
@@ -139,42 +147,38 @@ a git hook is already present.
 
 [Core team members](./CORE_TEAM.md) have the responsibility of pushing new releases to npm. The release process is as follows:
 
-1. Install `np` by Sindre Sorhus with `npm i -g np`. (note: you only have to do this once) `np` makes sure that everything is correct and runs tests and a build before it releases the new version for you.
-2. Make sure you have the latest changes and are on the master branch: `git checkout master && git pull origin master`
-3. Create a new branch based on the version number, for example `git checkout -b 3.4.1`
-4. Update the [CHANGELOG.md](./CHANGELOG.md) with the new version number, add a new Unreleased section at the top and edit the links at the bottom so everything is linked correctly
-5. Commit the Changelog changes with `git commit -m 'Update CHANGELOG'`
-6. Push the branch to the repo with `git push -u origin <branchname>`
-7. Run `np --any-branch` and follow its instructions
-8. Congratulations, you just published a new release of `styled-components`! :tada: Let everybody know on Twitter, in our community and all the other places
+1.Install all the dependecies by running `yarn` in the root folder. This will also install `lerna`.
```

Comment:
Let's recommend doing this after checking out to master and getting the latest changes, just in case the dependencies changed on the master branch!